text/0113-canonical-metadata.md

```diff
@@ -0,0 +1,625 @@
+            "$ref": "#/definitions/CSSFile"
+        }
+    },
+    "definitions": {
+        "HTMLTemplate": {
+            "type": "object",
+            "properties": {
+                "fileType": { "$ref": "#/definitions/FileType" },
+                "fileName": { "type": "string" },
+                "children": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/CustomElementReference"
+                    }
+                },
+                "staticResources": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/StaticResourceReference"
+                    }
+                },
+                "dynamicChildren": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/DynamicComponentReference"
+                    }
+                },
+                "directives": {
+                    "type": "object",
+                    "properties": {
+                        "forEach": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateForEachDirective" }
+                        },
+                        "iterator": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateIteratorDirective" }
+                        },
+                        "if": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateIfDirective" }
+                        },
+                        "lwcDom": {
+                            "type": "array", 
+                            "items": { "$ref" : "#/definitions/TemplateLwcDomDirective" }
+                        }
+                    }
+                }
+            }
+        },
+        "JSFile": {
+            "type": "object",
+            "properties": {
+                "fileType": { "$ref": "#/definitions/FileType" },
+                "fileName": { "type": "string" },
+                "componentClasses": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/Class"
+                    }
+                },
+                "imports": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Import"
+                    }
+                },
+                "dynamicImports": {
+                    "type": "array",
+                    "items": { 
+                        "$ref": "#/definitions/DynamicImport"
+                    }
+                },
+                "exports": {
+                    "type": "array",
+                    "items": [
+                        { "$ref": "#/definitions/Export"},
+                        { "$ref": "#/definitions/AggregatingExport"}
```

Comment:
What is `AggregatingExport` referencing to?

Reply:
It is referring to re-exporting. I could rename it to reexports if that is more intuitive.
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Re-exporting_Aggregating